.travis.yml

```diff
@@ -4,4 +4,4 @@ sudo: required
 dist: trusty
 before_install: ./maintainers/scripts/travis-nox-review-pr.sh nix
 install: ./maintainers/scripts/travis-nox-review-pr.sh nox
-script: ./maintainers/scripts/travis-nox-review-pr.sh build
+script: travis_wait 119 ./maintainers/scripts/travis-nox-review-pr.sh build
```

Comment:
It says that `travis_wait` just adds more output to prevent processes being killed when they don't print anything for a while. Is this what you wanted? Or is this just another change included in this PR?

https://docs.travis-ci.com/user/common-build-problems/#Build-times-out-because-no-output-was-received